src/OpenRiaServices.Hosting.AspNetCore/Test/OpenRiaServices.Hosting.AspNetCore.Test/OperationInvokerTests.cs

```diff
@@ -0,0 +1,237 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using Cities;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenRiaServices.Hosting.AspNetCore.Operations;
+using OpenRiaServices.Server;
+
+namespace OpenRiaServices.Hosting.AspNetCore;
+
+[TestClass]
+public class OperationInvokerTests
+{
+    private static CancellationTokenSource cts;
+    private static string submitType;
+    private static readonly DomainServiceDescription domainServiceDescription = DomainServiceDescription.GetDescription(typeof(OperationCanceledDomainService));
+    private static readonly SerializationHelper serializationHelper = new(domainServiceDescription);
+
+    [TestMethod]
+    public async Task TestCancelInMethodInvokeOperationInvoker()
+    {
+        cts = new CancellationTokenSource();
+        var operation = domainServiceDescription.GetInvokeOperation("CancelInvoke");
```

Comment:
I suggest you use nameof(...) for method name, that also makes it easy to see where method is used